src/parse.c

```diff
@@ -7394,8 +7465,13 @@ static void p_entity_declaration(tree_t unit)
    p_entity_header(unit);
    p_entity_declarative_part(unit);
 
+   imp_label_cnts.proc = 0;
+
    if (optional(tBEGIN))
       p_entity_statement_part(unit);
+   
+   // Hold number of implicitly labeled processes in current entity
+   tree_set_ival(unit, imp_label_cnts.proc);
```

Comment:
Don't store this in the AST. It's trivial to recompute by looping over the statements in the entity and counting how many start with `_`.

Reply:
I removed it and I do the search in `set_impl_label_proc_cnt`. Maybe the function could be generic with
something like: `get_stmts_matching_prefix`, and then accessing the counter in `p_architecture_body`,
but I wanted to avoid accessing directly the scope counter from `parse.c`